### PR TITLE
add ssh readme

### PR DESCRIPTION
**Pull request by wangcan0329**

### CODE REVIEW

File: job-tutorial/README.md [hunk 1]
```diff
@@ -225,3 +225,28 @@ A distributed TensorFlow job is listed below as an example:
 3. Submit the job through web portal
 
     Open web portal in a browser, click "Submit Job" and upload your config file.
+
+## Ssh Connection
```
Comment (YanjieGao) [inline job-tutorial/README.md]: refine "Ssh" to "SSH" ?

File: job-tutorial/README.md [hunk 2]
```diff
@@ -225,3 +225,28 @@ A distributed TensorFlow job is listed below as an example:
...
+   ```sh
+   chmod 400 application_id
+   ```
+5. Use `ssh` command to connect into container. for example
+   ```sh
+   ssh -i application_id -p sshPort root@container_ip
+   ```
+### Ssh connect inside containers
+
+You can use `ssh $PAI_CURRENT_TASK_ROLE_NAME-$PAI_CURRENT_TASK_ROLE_CURRENT_TASK_INDEX` command to connect into another containers which belong to the same job.
```
Comment (sterowang) [inline job-tutorial/README.md]: It's better to add an example command line here.

File: job-tutorial/README.md (hunk 1 above)
Comment (wangcan0329) [reply job-tutorial/README.md]: Sure. I will refine the words

File: job-tutorial/README.md (hunk 2 above)
Comment (wangcan0329) [reply job-tutorial/README.md]: Sure. I will add an example

File: job-tutorial/README.md [hunk 3]
```diff
@@ -225,3 +225,31 @@ A distributed TensorFlow job is listed below as an example:
...
+   ```sh
+   wget http://host:port/webhdfs/v1/Container/userName/jobName/ssh/application_id/.ssh/application_id?op=OPEN -O application_id
+   ```
+4. Use `chmod` command to set correct permission for the key file.
+   ```sh
+   chmod 400 application_id
+   ```
+5. Use `ssh` command to connect into container. for example
+   ```sh
+   ssh -i application_id -p sshPort root@container_ip
```
Comment (abuccts) [inline job-tutorial/README.md]: use `ssh_port` to be consistent?

File: job-tutorial/README.md [hunk 4]
```diff
@@ -225,3 +225,31 @@ A distributed TensorFlow job is listed below as an example:
...
+   ```
+### SSH connect inside containers
+
+You can use `ssh $PAI_CURRENT_TASK_ROLE_NAME-$PAI_CURRENT_TASK_ROLE_CURRENT_TASK_INDEX` command to connect into another containers which belong to the same job. For example, if there are two taskRoles: master and worker, you can connect to worker0 container directly with below command line:
```
Comment (abuccts) [inline job-tutorial/README.md]: `worker-0` ?

**Pull request merged**